src/test/java/net/minecraftforge/debug/client/AudioStreamTest.java

```diff
@@ -0,0 +1,91 @@
+package net.minecraftforge.debug.client;
+
+import net.minecraft.client.Minecraft;
+import net.minecraft.client.resources.sounds.AbstractSoundInstance;
+import net.minecraft.client.resources.sounds.Sound;
+import net.minecraft.client.sounds.AudioStream;
+import net.minecraft.client.sounds.SoundBufferLibrary;
+import net.minecraft.resources.ResourceLocation;
+import net.minecraft.sounds.SoundSource;
+import net.minecraftforge.client.event.ClientChatEvent;
+import net.minecraftforge.common.MinecraftForge;
+import net.minecraftforge.fml.common.Mod;
+import net.minecraftforge.fml.loading.FMLLoader;
+import org.jetbrains.annotations.NotNull;
+import org.lwjgl.BufferUtils;
+
+import javax.sound.sampled.AudioFormat;
+import java.nio.ByteBuffer;
+import java.util.concurrent.CompletableFuture;
+
+@Mod(AudioStreamTest.MOD_ID)
+public class AudioStreamTest
+{
+    private static final boolean ENABLED = true;
+    static final String MOD_ID = "audio_stream_test";
+
+    public AudioStreamTest()
+    {
+        if (ENABLED && FMLLoader.getDist().isClient())
+        {
+            MinecraftForge.EVENT_BUS.addListener(this::onClientChatEvent);
+        }
+    }
+
+    public void onClientChatEvent(ClientChatEvent event)
+    {
+        if (event.getMessage().equalsIgnoreCase("sine wave"))
+        {
+            event.setCanceled(true);
+            Minecraft.getInstance().getSoundManager().play(new SineSound());
```

Comment:
This test mod fails to load on the dedicated server because of the `SoundInstance` client-only class. Please fix this test mod so it does not cause a mod loading error due to classloading on the dedicated server.